ruby/src/main/java/com/google/protobuf/jruby/RubyMessage.java

```diff
@@ -246,16 +246,15 @@ public IRubyObject methodMissing(ThreadContext context, IRubyObject[] args) {
     public IRubyObject dup(ThreadContext context) {
```

Comment:
np!  

I'll throw a few more PRs your way, and then once that stabilizes, I'll take a stab at the test suite.  Because their is a lot of hand-rolled functionality rather than using the ruby stdlib or core (for performance and strong typing), I think tests which are usually not done (like testing `push`) need to be done.  I'm happy to take a stab at that in the next few weeks.